server/routes/admin.js

```diff
@@ -1,20 +1,54 @@
-var express = require('express');
-var router = express.Router();
+const express = require('express');
+const router = express.Router();
 const categoryClient = require('../dbClients/categoriesDB');
 const articleClient = require('../dbClients/articlesDB')
- 
+const ObjectId = require('mongodb').ObjectID;
 
 router.get("/", function (req, res, next) {
   res.send('admin');
 });
 
 router.get('/categories', function (req, res, next) {
   const callback = (error, category) => {
-     res.render("admin-list-categories",{
+    res.render("admin-list-categories", {
       category
-     }) 
+    })
+  }
+  categoryClient.findCategories({},callback);
+});
+
+router.get('/articles', function (req, res, next) {
+  const callback = (error, articles) => {
+    res.render("admin-list-articles", {
+      articles: articles
+    })
+  }
+  articleClient.findArticles({}, callback);
+});
+ 
+router.get('/categories/edit/:categoryId', (req, res) => {
+  let categoryId = req.params.categoryId;
+  const callback = (error, category) => {
+    res.render("admin-edit-category", {
+      category: category[0]
+    })
+  }
+  categoryClient.findCategories({ "_id": ObjectId(categoryId) }, callback);
```

Comment:
you shouldn't need to use ObjectId - in DBClient, you should use `findById` instead of find. Look here http://mongoosejs.com/docs/api.html#model_Model.findById